src/partitioned-dbs/index.rst

```diff
@@ -0,0 +1,384 @@
+this precise problem.
+
+
+What is a partition?
+====================
+
+In the previous section, we introduced a hypothetical database that contains
+sensor readings from an IoT field monitoring service. In this particular
+use case, it's quite logical to group all documents by their ``sensor_id``
+field. In this case, we would call the ``sensor_id`` the partition.
+
+A good partition has two basic properties. First, it should have a high
+cardinality. That is, there is a large number of values for the partition.
+A database that has a single partition would be an anti-pattern for this
+feature. Secondly, the amount of data per partition should be "small". The
+general recommendation is to limit individual partitions to less than ten
+gigabytes of data. Which, for the example of sensor documents, equates to roughly
+60,000 years of data.
+
+
+Why use partitions?
+===================
+
+The primary benefit of using partitioned databases is for the performance
+of partitioned queries. Large databases with lots of documents often
+have a similar pattern where there are groups of related documents that
+are queried together often.
+
+By using partitions, we can execute queries against these individual groups
+of documents more efficiently by placing the entire group within a specific
+shard on disk. Thus, the view engine only has to consult one copy of the
+given shard range when executing a query instead of executing
+the query across all ``Q`` shards in the database.
+
+
+Partitions By Example
+=====================
+
+To create a partitioned database, we simply need to pass a query string
+parameter.
+
+.. code-block:: bash
+
+    shell> curl -X PUT http://127.0.0.1:5984/my_new_db?partitioned=true
+    {"ok":true}
+
+To see that our database is partitioned, we can look at the database
+information:
+
+.. code-block:: bash
+
+    shell> curl http://127.0.0.1:5984/my_new_db
+    {
+      "cluster": {
+        "n": 3,
+        "q": 8,
+        "r": 2,
+        "w": 2
+      },
+      "compact_running": false,
+      "data_size": 0,
+      "db_name": "my_new_db",
+      "disk_format_version": 7,
+      "disk_size": 66784,
+      "doc_count": 0,
+      "doc_del_count": 0,
+      "instance_start_time": "0",
+      "other": {
+        "data_size": 0
+      },
+      "props": {
+        "partitioned": true
+      },
+      "purge_seq": "0-g1AAAAFDeJzLYWBg4M...",
+      "sizes": {
+        "active": 0,
+        "external": 0,
+        "file": 66784
+      },
+      "update_seq": "0-g1AAAAFDeJzLYWBg4M..."
+    }
+
+
+You'll now see that the ``"props"`` member contains ``"partitioned": true``.
+
+.. note::
+
+    The format for document ids in a partitioned database is
+    ``partition:docid``. Every regular document (i.e., everything
+    except design and local documents) in a partitioned database
+    must follow this format.
+
+.. note::
+
+    System databases are *not* allowed to be partitioned. This is
```

Comment:
"System databases (such as `_users`) are *not*..."

Reply:
Addressed in 347a6ce.